packages/sampler/stories/qa-stories/Scroller.js

```diff
@@ -58,7 +60,7 @@ storiesOf('Scroller', module)
 		'Horizontal scroll',
 		() => (
 			<Scroller
-				direction={select('direction', prop.direction, Scroller, 'horizontal')}
+				direction={select('direction', prop.direction, Scroller, 'vertical')}
```

Comment:
Is there a reason you changed this default?

Reply:
Removed direction knob now